### PR TITLE
3 openshift

### PR DESCRIPTION
**Pull request by adberglund**
This should let us spin up masu in isolation (with its own copy of a DB) for local dev work, and put it into the project with koku already running

Ex. 

With openshift running, but no koku app

`make oc-dev-all` would spin up masu + a db.

If openshift is running and koku + db already exist

`make oc-create-masu` should put it in the `myproject` project where koku is deployed.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,6 +4,8 @@ branch = True
 source = masu
 omit =
   *config.py
+  tests/*
```
Comment (chambridge) [inline .coveragerc]: Maybe:
```
*test/*
*/tests_*.py
```

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,101 @@
+PYTHON	= $(shell which python)
+
+TOPDIR  = $(shell pwd)
+PYDIR	= koku
+APIDOC = apidoc
```
Comment (chambridge) [inline Makefile]: don't think we need this here.

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,101 @@
+PYTHON	= $(shell which python)
+
+TOPDIR  = $(shell pwd)
+PYDIR	= koku
```
Comment (chambridge) [inline Makefile]: `masu`?

Comment (adberglund) [reply Makefile]: 😴 @ the ☸️ again, clearly. Thanks for catching these.

**Pull request merged**